### ISSUE TITLE
The Road to 1.0

### ISSUE THREAD

**Issue by alefranz** [alefranz/MELT#40]
A non exhaustive things to consider for v1:

- ~Complete major refactoring~ Post 1.0
- [x] Drop support for targets and libraries that have reached end of life #44
- ~Consider transforming MELT.Xunit into MELT.Assert, with no explicit reference to xunit.assert~ Post 1.0
- ~Consider including LoggingAssert as part of the main package or turning the MELT main package into a meta package that includes MELT.Core and MELT.Assert~ Post 1.0
- [x] ~Release a last 0.x version with~ deprecation warnings to help migrate (in 1.0)
- [x] Refresh branding #43

**Comment by 304NotModified** [comment]
Hi @alefranz 

As this is not really in movement, I propose the following

- "Complete major refactoring" -> do this for 2.0
- "Drop support for targets and libraries that have reached end of life" OK
- "Consider transforming MELT.Xunit into MELT.Assert, with no explicit reference to xunit.assert" maybe 2.0
- "Consider including LoggingAssert ..." maybe 2.0\
- "Refresh branding" not really needed?

So in summary, drop old EOL packages and move 0.9 to 1.0 :)

**Comment by alefranz** [comment]
I've started working on it, thanks for the gentle nudge 🙂

**Comment by 304NotModified** [comment]
Cool!

**Comment by alefranz** [comment]
@304NotModified Created [1.0 RC 2](https://github.com/alefranz/MELT/releases/tag/v1.0.0-rc.2), I will test more, then release.

**Comment by 304NotModified** [comment]
Great news!

**Comment by alefranz** [comment]
🥳 [Release 1.0](https://github.com/alefranz/MELT/releases/tag/v1.0.0) is NOW available!

Any problem or feature request, please raise an issue!

**Comment by 304NotModified** [comment]
Thx & Congratulations @alefranz!
